azure-pipelines.yml

```diff
@@ -124,7 +124,7 @@ jobs:
       solution: build/snmalloc.sln
       msbuildArguments: '/m /p:Configuration=$(BuildType)'
 
-  - script: 'ctest -j 4 --interactive-debug-mode 0 --output-on-failure -C $(BuildType)'
+  - script: 'ctest -j 1 --interactive-debug-mode 0 --output-on-failure -C $(BuildType)'
```

Comment:
This change now causes a conflict.  I turned this down to -j2 on the Win8Compat variant but left it at -j4 elsewhere.  It seemed to make the tests happy again...